tests/e2e/utils/StringUtil.ts

```diff
@@ -59,4 +59,32 @@ export class StringUtil {
 
 		return command.replace(/[{}]/g, '').replace(/(?<!")\${?[a-zA-Z0-9_+\-\s]+\b}?/gm, '"$&"');
 	}
+
+	/**
+	 * replaces the cookie value of the specified cookie
+	 * @param cookie cookie names and values, seperated with ;
+	 * @param name name of cookie to replace its value for
+	 * @param replaceStr the new value of the cookie
+	 * @return updated cookie string with the cookie value replaced
+	 */
+	static updateCookieValue(cookie: string, name: string, replaceStr: string): string {
```

Comment:
@nallikaea I have made changes to use `StringUtil`

Reply:
Thank you!